usecases/Concrete/tests/test_generate_metadata_from_emodul_BAM.py

```diff
@@ -0,0 +1,124 @@
+
+import unittest
+import os
+from pathlib import Path
+from datetime import datetime
+
+class test_generate_metadata_from_emodul_BAM(unittest.TestCase):
+    
+    # read specimen.dat file to make sure that headers work correctly
+    def read_specimen_file(self, file):
+        with open(file) as specimen:
+            print ('test')
+            #lines = specimen.readlines()
+            # TODO: if checking line by line is needed it has to change
+            for i, line in enumerate(specimen):
+                if (specimen.name.__contains__('BA Los M VI-6')):
+                    if ("Bediener" in line):
+                        date_val = line.split('\t')
+                        origin_date = date_val[9]
+                        str_date = ''.join(origin_date.split())
+                        str_date_time = datetime.strptime(str_date, '%d.%m.%Y%H:%M:%S')
+                        self.assertIsInstance(str_date_time, datetime)
+                        self.assertListEqual([str_date_time.year, str_date_time.month, str_date_time.day, 
+                        str_date_time.hour, str_date_time.minute, str_date_time.second], 
```

Comment:
What are you testing here? The result that you compare with should not be obtained by reading in the file again, but take the result that is processed in the metadata extraction functions (those are the ones we would like to test).